release-notes/2023-04/README.md

```diff
@@ -0,0 +1,227 @@
+# SystemLink Enterprise 2023-04 Release Notes
+
+The 2023-04 release for SystemLink Enterprise has been published to <https://niedge01.jfrog.io>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## Upgrading from the 2023-03 to the 2023-04 release
+
+JupyterHub user pod PVCs incorporate a user's email address in the PVC name. This aids in the association of users to JupyterHub PVCs so unused PVCs can be safely removed. Users will lose access to their personal Jupyter notebooks and other settings in their JupyterHub instance due to this change. Users are encouraged to backup their personal Jupyter notebooks locally prior to upgrading. Old PVCs are not automatically deleted and can be recovered. You can opt-out of this new behavior. This change does not affect notebooks [published](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/sharing-a-jupyter-notebook.html) to SystemLink Refer to **`sl-jupyterhub 1.0.0` PVC name change** in **Helm Chart Breaking Changes** for details.
+
+## New Features and Behavior changes
+
+- You can use data spaces to find and visualize a subset of your test result data. This allows you to interactively slice data to diagnose issues or discover trends that may not obvious without visual inspection.
```

Comment:
> of your test result data

Do we need to clarify this is Test Monitor test data? "Test result data" could refer to data tables just as easily as it could refer to Test Monitor data.

Reply:
Good point. I've clarified